content/en/seachat/getting-started/02-quick-start.md

```diff
@@ -0,0 +1,143 @@
+<br/>
+
+<br/>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step2-2.jpg" alt="">
+<br/>
+
+<br/>
+
+> :bulb: **Note** 
+>
+>  The live agent feature will require a human agent in case the customer wishes to be transferred to one. The customer will then have the option to request a human agent during the chat.
+> 
+
+## Step 3:  Add Knowledge
+To fully leverage the capabilities of your SeaChat agent, we need to add knowledge to its knowledge base. SeaChat offers a range of options for uploading content to your agent's knowledge base. Find your agent's knowledge base by navigating to the **Knowledge Base** dashboard under **Agent Configuration** in the sidebar menu. 
+
+Feel free to try out the different ways to upload data to your knowledge base. For the purposes of this tutorial, we'll focus on utilizing the spreadsheet upload method. However, if you're interested in exploring additional data uploading techniques, we invite you to consult the [Add Knowledge]( /en/seachat/seachat-intro/03-add-knowledge/add-knowledge-intro) section within our user manual for comprehensive instructions and tips.
+<br/>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step3.jpg" alt="">
+<br/>
+
+## Step  4:  Upload Files to Knowledge Base
+Uploading a spreadsheet is as simple as clicking on the upload icon and choosing the desired spreadsheet. Once you see your uploaded files in the preview section below the drag-and-drop section, click on the **Next** button to confirm. 
+Now, ncc_Car_Parks (1) is uploaded, adn we can see each header and its corresponding data in the preview section. My Parking Agent is now ready for the next step.
+
+<br/>
+
+<a href="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step4.jpg" target="_blank">
+<img width="80%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step4.jpg" alt="">
+</a>
+<br/>
+
+<br/>
+
+
+To see the parking lot data uploaded, we simply navigate to the **Existing** section in the top-right corner of the dashboard once your files have been successfully uploaded.
+
+<br/>
+<a href="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step4-2.jpg" target="_blank">
+<img width="80%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step4-2.jpg" alt="">
+</a>
+<br/>
+
+<br/>
+
+That’s it. Now, My Parking Agent has processed the uploaded data, and let’s set it up for a test to see if it needs further fine-tuning. Click on the **Test AI Agent** button located in the bottom-right corner. Test away and let’s fine-tune the AI agent for optimal performance!.
```

Comment:
the user might not understand why he needs to test and fine-tine the AI agent.

Try to explain the concept a bit more. 
"Before launch, let's test the AI agent's accuracy based on the uploaded knowledge. This process is called fine-tuning. Simply chat with the agent to assess its responses, and make any necessary adjustments to the knowledge base for better results.